index.html

```diff
@@ -17,12 +18,27 @@
 <body>
 
  <!-- navigation code here -->
+<section class="navBar">
+      <h1>SpaceWalkers</h1>
+      
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+        <a href="#">Sign In</a>
+      </nav>
+    </section>
 
   <div class="container home">
 
+    
+
     <header class="intro">
        <!-- image code here -->
-
+       <div class="headerImg">
+       <img src="C:\Users\Toosd\Downloads\Lambda\Sprint-Challenge--Advanced-CSS\img\main-header.png" class="headerImg" alt="ClearSkyHeader" style="width: 100%;">
```

Comment:
image isn't showing up remotely because it is looking inside your computer for the file. make sure to route links from file you're in